tsdb/engine/tsm1/engine.go

```diff
@@ -44,8 +44,8 @@ import (
 // to support adding templated data from the command line.
 // This can probably be worked into the upstream tmpl
 // but isn't at the moment.
-//go:generate go run ../../../tools/tmpl -i -data=file_store.gen.go.tmpldata file_store.gen.go.tmpl=file_store.gen.go
```

Comment:
Why did we need the change here and in `file_store.gen.go.tmpl`?

Reply:
`-i` says to run `goimports` on the generated code.

We don't actually need `goimports` (the change inside `file_store.gen.go.tmpl` below makes us have the right imports) and the `goimports` is just whatever binary happens to be on your path, which is annoying when running the codegen for the first time if you don't happen to have `goimports` on your PATH at all.

Reply:
Makes sense thanks 👍